packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -1297,6 +1297,30 @@ export default class Dashboard<M extends Meta, B extends Body> extends UIPlugin<
     }
   }
 
+  #getStatusBarOpts() {
+    return {
+      hideUploadButton: this.opts.hideUploadButton,
+      hideRetryButton: this.opts.hideRetryButton,
+      hidePauseResumeButton: this.opts.hidePauseResumeButton,
+      hideCancelButton: this.opts.hideCancelButton,
+      showProgressDetails: this.opts.showProgressDetails,
+      hideAfterFinish: this.opts.hideProgressAfterFinish,
+      locale: this.opts.locale,
+      doneButtonHandler: this.opts.doneButtonHandler,
+    }
+  }
+
+  setOptions(opts: DashboardOptions<Meta, Body>) {
+    super.setOptions(opts)
+    this.uppy
+      .getPlugin(this.#getStatusBarId())
+      ?.setOptions(this.#getStatusBarOpts())
```

Comment:
Should we do the same for thumbnail generator?

Reply:
good catch